### ISSUE TITLE
Unable to Download Weight File

### ISSUE THREAD

**Issue by Qianer-Li** [hustvl/Snap-Snap#5]
I encountered an issue while attempting to download the weight file. I noticed that the link to the weight file points to your recycle bin, which prevents me from downloading it normally. Could you please restore the weight file from the recycle bin, or provide an alternative usable download link? Thank you very much.

**Comment by Qianer-Li** [comment]
And, I am using the train_dust3r.py code, and I modified the line data, loss = self.model.forward_dust3r_wo_side(data) to data, loss = self.model.forward_dust3r(data) for training. However, I noticed that the generated dust3r.pth file seems to be of poor quality. Could you tell me what the problem might be? Thank you for your answer.

**Comment by sai-pavan-tej** [comment]
Im also facing the same issue, weight are not available. @lujzz  can you pls provide the weights file
